frontend/src/scenes/surveys/surveyViewViz.tsx

```diff
@@ -460,7 +462,14 @@ export function OpenTextViz({
                 <></>
             ) : (
                 <>
-                    <div className="font-semibold text-muted-alt">Open text</div>
+                    <Tooltip title="See all Open Text responses in the Events table at the bottom.">
+                        <div className="inline-flex gap-1">
+                            <div className="font-semibold text-muted-alt">Open text</div>
+                            <LemonDivider vertical className="my-1 mx-1" />
+                            <div className="font-semibold text-muted-alt">random responses</div>
```

Comment:
```suggestion
                            <div className="font-semibold text-muted-alt">random selection</div>
```

something like that sounds a bit better than random responses because it's the selection really right 😅

Reply:
good point! 😄